cmd/common-main.go

```diff
@@ -339,6 +339,12 @@ func handleCommonEnvVars() {
 		os.Unsetenv(config.EnvRootUserOld)
 		os.Unsetenv(config.EnvRootPasswordOld)
 	}
+	if tiers := env.Get("_DEBUG_REMOTE_TIERS_IMMEDIATELY", ""); tiers != "" {
```

Comment:
can we have the env var name start with `_MINIO_` inline with our regular env var convention?